frontend/src/metabase/query_builder/actions/core/core.js

```diff
@@ -264,9 +264,10 @@ export const apiUpdateQuestion = (question, { rerunQuery } = {}) => {
 
     dispatch({ type: API_UPDATE_QUESTION, payload: updatedQuestion.card() });
 
+    const metadataOptions = { reload: question.isDataset() };
```

Comment:
`rerunQuery` will be `false` when updating the model, but we still need to load metadata when the question is updated.